packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -341,6 +341,11 @@ const TRANSFORMS = {
     // See https://xcp-ng.org/forum/topic/4810
     const addresses = {}
     for (const key in networks) {
+      // Some fields may be emtpy
+      // See https://xcp-ng.org/forum/topic/4810/netbox-plugin-error-ipaddr-the-address-has-neither-ipv6-nor-ipv4-format/27?_=1658735770330
+      if (networks[key] === '') {
```

Comment:
should we trim it to also handle  spaces  ? can this case append ?

Reply:
I don't know if it can happen but it's safer to do it anyway.